test/fuzz/transformation_replace_id_with_synonym_test.cpp

```diff
@@ -1800,7 +1800,7 @@ TEST(TransformationReplaceIdWithSynonymTest, IncompatibleTypes) {
 //  should be updated to cover more atomic operations, and once the issue is
 //  fixed the test should be enabled.
 TEST(TransformationReplaceIdWithSynonymTest,
```

Comment:
The TODO comment above this test should be removed, because you have now addressed the TODO.

Reply:
Done.